Tools/gazebo_sitl_multiple_run.sh

```diff
@@ -47,27 +53,31 @@ then
 	exit 1
 fi
 
-while getopts n:m:w:s: option
+while getopts n:m:w:s:t: option
 do
 	case "${option}"
 	in
 		n) NUM_VEHICLES=${OPTARG};;
 		m) VEHICLE_MODEL=${OPTARG};;
 		w) WORLD=${OPTARG};;
 		s) SCRIPT=${OPTARG};;
+		t) TARGET=${OPTARG};;
 	esac
 done
 
 num_vehicles=${NUM_VEHICLES:=3}
-export PX4_SIM_MODEL=${VEHICLE_MODEL:=iris}
 world=${WORLD:=empty}
-echo ${SCRIPT}
-
+target=${TARGET:=default}
+if [ $TARGET == "rtps" ] && [ -z $VEHICLE_MODEL ] ; then
+	VEHICLE_MODEL=iris_rtps
+fi
```

Comment:
I understand the intention of adding this, but I am not sure if this case handling is necessary.

Reply:
I want to clarify the option like default iris model for default target.

Reply:
Yes exactly, and the default is `iris` which is already handled. Therefore IMHO we dont need another default

Reply:
OK. I remove it :-)